src/app/job-list/job-list-popup/job-list-popup.component.ts

```diff
@@ -131,21 +135,30 @@ export class JobListPopupComponent implements OnInit {
     ];
   }
   createExpressions(data: any) {
+    for (let i=0; i < this.clinicianModel.length; i++){
     for (let index = 0; index < data.length; index++) {
+      if(this.clinicianModel[i].name==data[index].name){
       this.cliniciansRelationshipRowData.push({ Role: data[index].name, Expression : data[index].expressions});
-    }
+      }
+    }}
   }
   
   createClinicianEfficieny(data: any) {
+    for (let i=0; i < this.clinicianModel.length; i++){
```

Comment:
@GowsalyaSG Why we introduce new for loop?

Reply:
we are displaying the details orderly based on default clinician model's order